storybook/storybook-docs/src/icon-gallery.stories.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta } from "@storybook/blocks";
+import { AmsterdamIconGallery } from "./components/AmsterdamIconGallery";
+
+<Meta title="Docs/Designrichtlijnen/Iconen" />
```

Comment:
An overview of the icons isn't really a 'designrichtlijn', right? Shouldn't we show them directly under docs?

Reply:
I never really understood the title ‘Design rightlijnen’. If we rename it to ‘Ontwerpkeuzes’ or ‘Vormgeving’ or ‘Digitale huisstijl’, everything we currently have fits again.

Reply:
The idea behind this section is to define the rationale behind design choices, which flows into the actual design. So something like: our interactive color is blue, which results in our links being blue. 

Something like 'Ontwerpkeuzes' would work as well, but 'Vormgeving' means something else imo. Our _vormgeving_ is the result of our _ontwerpkeuzes_.

In the same way, our icons are the result of our _ontwerpkeuzes_, not the _ontwerpkeuze_ itself.

Reply:
Alright, I moved icons into a new ‘assets’ directory and the parts of the typography dealing with the font itself as well.